contracts/interfaces/IAccount.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity >=0.7.5;
```

Comment:
interfaces are deliberately ">=" and not "^". That is, we allow any version (That supports ABIv2 encoding). we don't want to change them anymore, even when compiler changes to 0.9 or 1.0

Reply:
I'm okay with having a wide range for interfaces. Not necessary as there are many alternative repos for these interface declarations now.

Reply:
So why not remove this requirement from interfaces altogether?